.gitignore

```diff
@@ -277,3 +277,5 @@ src/frontend/temp
 
 .dspy_cache/
 *.db
+
+.vscode
```

Comment:
Not sure if you want to add this here - maybe up at [line 16](https://github.com/langflow-ai/langflow/pull/7052/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R16)

Reply:
The `.vscode` folder is part of the repo.